### PR TITLE
refactor(ci): change latest stable branch and schedule for QA run

### PR DESCRIPTION
**Pull request by pihme**
## Description

This changes the latest stable branch to `stable/1.1` and assigns it a time slot for QA run.

## Definition of Done

Code changes:
* [X] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

**Comment by pihme** [comment]
@korthout 
* Since you were the release manager - was there any mention of making this change in one of the release processes?
* Can I "backport" this change to develop, or would that just end in a mess?

**Comment by korthout** [comment]
> was there any mention of making this change in one of the release processes?

No, we should add it to the process as well as to the wiki. Ping @npepinpe to add this to the wiki

> * Can I "backport" this change to develop?

Feel free to "forwardport" it 😄

File: Jenkinsfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ def masterBranchName = 'master'
 def isMasterBranch = env.BRANCH_NAME == masterBranchName
 def developBranchName = 'develop'
 def isDevelopBranch = env.BRANCH_NAME == developBranchName
-def latestStableBranchName = 'stable/1.0'
+def latestStableBranchName = 'stable/1.1'
```
Comment (korthout) [inline Jenkinsfile:13]: Can we use wildcard markers in here?  Like `stable/1.*` or like a regex `stable/1\.[0-9]+`

**Review by korthout — APPROVED** [review]
LGTM 👍

File: Jenkinsfile (hunk 1 above)
Comment (pihme) [reply Jenkinsfile:13]: I would rather not, this would assign all wildcard matches the same CRON schedule.

**Comment by pihme** [comment]
bors retry

**Comment by pihme** [comment]
bors retry

**Comment by npepinpe** [comment]
FYI, no need to backport to 0.26 anymore as it was phased out. Did you mean to backport to 1.0?

**Comment by pihme** [comment]
No, it was to disable the nightly builds in stable/0.26. We removed the 0.26 testbench, but so far we didn't stop the nightly builds.

**Pull request merged**